### DISCUSSION TITLE
EP Installed on 2 Machines 24H2  OS 2611.1742

### DISCUSSION THREAD

**Discussion by christopbader** [valinet/ExplorerPatcher#3759] [General]
Waited long enough so updated to 24H2, installed EP on my 2 machines and have no issues, I followed carefully the instructions shown on EPs page, put files into Virus exception list and seems to be running just fine even the EP windows 10 taskbar is working well, Great job to the Developers.
![Winver](https://github.com/user-attachments/assets/a8782159-4189-43b5-ab2b-af32c47f92a5)

**Comment by BaliB4ba** [comment]
Are you seeing the Weather entry on Settings?

**Reply by christopbader** [reply]
![Screenshot (19)](https://github.com/user-attachments/assets/1c73429a-884b-4cd7-862f-f9b66d7ec1f6)
No, there's no weather entry on EP and i never noticed it till now, I'm not too worried about the weather entry

**Comment by pyrates999** [comment]
weather is not yet supported on the windows 10 (ExplorerPatcher) taskbar: https://github.com/valinet/ExplorerPatcher/issues/3546

**Reply by christopbader** [reply]
Yeah I love EP, everything i need in one program, i know others have had issues with EP but I've haven't experienced any, and the weather entry??  who cares

**Comment by kai-uwe-rommel** [comment]
I just updated my PC to 24H2 26100.1742 and updated EP to the latest release 22621.3880.66.6 but the Windows 10 taskbar simply does not work. No error during installation of EP and no AV kicking in. We use Symantec EP for AV and it was the same before with 23H2 where EP works fine. I can open the EP settings from the start menu (it does not show in the context menu of the taskbar) and there under taskbar it says "The Windows 10 taskbar is not available in this Windows version". 

Am I doing anything wrong? Is there a way to solve the issue?

I updated another PC in the office with the same versions and there it also fails. So it is reproducible for me.

**Reply by christopbader** [reply]
![Screenshot (21)](https://github.com/user-attachments/assets/7856a4d2-b75e-4dc9-83ed-4ab06adc34c0)

**Comment by kai-uwe-rommel** [comment]
Ah, now I understand! There is the old "Windows 10" mode and a new "Windows 10 (ExplorerPatcher)" which I need to use because only this one works with 24H2. I had read about this but thought it would simply replace the original mode directly and therefore didn't even click yet on this setting ... sorry. Problem solved. Thanks!

**Reply by christopbader** [reply]
I think its a great thing to have the Win-10 taskbar option since Microsoft removed the code for the win-10 taskbar which made it impossible to activate, so the developers of EP made their own code and incorporated it into EP, We now have two options for the taskbar, Win-11 or Win-10 style and we can thank the Developers
